.licenseupdater.yaml

```diff
@@ -0,0 +1,17 @@
+licenses: ["BSL", "RCL"]
+ignore:
+  - extension: .yaml
+    match: /test(s){0,1}/
+  - match: generated
+files:
+  - name: licenses/boilerplate.go.txt
+    type: go
+    license: BSL
+matches:
+  - extension: .yaml
+    directory: src/go/k8s/helm-chart/charts/redpanda-operator/templates
```

Comment:
That path should be removed from this repo as operator helm chart is now hosted https://github.com/redpanda-data/helm-charts/tree/main/charts/operator

Reply:
Do you know if the chart here is completely unused (not just in tests, but altogether? Like we don't publish this anywhere, right?). If so, maybe I can take a second pass and just remove that entire directory?